### ISSUE TITLE
[Firefox] preventDefaultOnInputTarget 

### ISSUE THREAD

**Issue by ricohumme** [fomantic/Fomantic-UI#543]
# Bug Report
Hi,

I just built Fomantic UI from the develop branch because I was waiting on an important fix.
It has just now come to my intention that Chrome works just fine with but when using Firefox, i somehow get the following error: `jQuery.Deferred exception: event is not defined preventDefaultOnInputTarget`

I found that the following piece of code was causing this:
`Source: semantic.min.js`

```javascript 
preventDefaultOnInputTarget: function () {
    event && S(event.target).is(p.input) && (m.verbose('Preventing default check action after manual check action'), event.preventDefault())
}
```
Now, as you can see, something is being done to `event`, but it is not defined with the scope.
I looked up further in the code and saw that this happens in `checkbox`

With release 2.7.2, there were no issues reported, so I'm assuming this is somehow related to the upcoming release.

**Comment by prudho** [comment]
Related to 00191d266509575389f52f91875567039578fe06. Let's summon @lubber-de here :)

**Comment by lubber-de** [comment]
Already fixed by #534

**Comment by dutrieux** [comment]
Maybe related with this one : https://github.com/fomantic/Fomantic-UI/issues/533

**Comment by lubber-de** [comment]
Mmh.. Maybe not... I'll check

**Comment by lubber-de** [comment]
Fixed by #544 
Thank you very much for catching this!
I am very sorry for this and used Browserstack once again to verify it works now... twice 🙂

**Comment by ricohumme** [comment]
No problem :)
Thanks for applying the fix so quickly!

**Comment by ricohumme** [comment]
@hammy2899 No clue as to why this should remain open as the resolving element resides in another issue (#544)

**Comment by hammy2899** [comment]
@ricohumme This issue brings to light something else as shown in @lubber-de's PR. We keep issues open util their relevant PR is merged into master.
